frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -56,7 +56,7 @@ const Details_ = ({flags, obj: pvc}) => {
   const volumeName = _.get(pvc, 'spec.volumeName');
   const requestedStorage = _.get(pvc, 'spec.resources.requests.storage');
   const storage = _.get(pvc, 'status.capacity.storage');
-  const accessModes = _.get(pvc, 'status.accessModes');
+  const accessModes = _.get(pvc, 'spec.accessModes');
```

Comment:
If we only show one, I think we should be showing the actual access modes vs requested.